# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/popt-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- popt-1.7/Makefile.in	2002-08-31 13:22:18.000000000 +0000
+++ popt-1.7/Makefile.in.orig	2005-05-12 18:12:40.000000000 +0000
@@ -164,7 +164,7 @@
 LTLIBRARIES = $(lib_LTLIBRARIES)
 
 libpopt_la_LDFLAGS =
-libpopt_la_LIBADD =
+libpopt_la_LIBADD = -lintl
 am_libpopt_la_OBJECTS = popt.lo findme.lo poptparse.lo poptconfig.lo \
 	popthelp.lo
 libpopt_la_OBJECTS = $(am_libpopt_la_OBJECTS)
